test/NuGet.Clients.Tests/NuGet.VisualStudio.Common.Test/Telemetry/PackageSourceTelemetryTests.cs

```diff
@@ -16,6 +16,20 @@ namespace NuGet.VisualStudio.Common.Test.Telemetry
 {
     public class PackageSourceTelemetryTests
     {
+        [Fact]
+        public void Ctor_DuplicateSources_DoesNotThrow()
+        {
+            // Arrange
+            var source = new PackageSource("https://source.test/v3/index.json");
```

Comment:
Sounds like it's currently irrelevant whether two distinct Names have the same URL because we are only reporting telemetry by the URL. No other metadata in PackageSource is relevant. If the protocol is different between two PackageSources with the same URL, one will fail by nature of the Server supporting the wrong protocol.